portable_fs/template/fs.py

```diff
@@ -236,10 +236,12 @@ def get_spec_handler(self, resource):
         kwargs = resource['resource_kwargs']
         rpath = resource['resource_path']
         root = resource.get('root', '')
-
+        root = self.root_map.get(root, root)
         if root:
-            root = self.root_map.get(root, root)
+            if os.path.isabs(rpath):
+                rpath = rpath[1:]
```

Comment:
That is potentially true. I will look into os.path for a way to remove the drive/root